### ISSUE TITLE
Paginator component page selection

### ISSUE THREAD

**Issue by f3rran** [primefaces/primevue#7023]
### Describe the bug

I'm trying to use the Paginator component to paginate a list with query param for the page but I can not see where to pass the page number to the component so it reflects the right page selected.

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primevue.org/paginator/

### Environment

Laravel + Inertia

### Vue version

3.3.13

### PrimeVue version

4.2.2

### Steps to reproduce the behavior

Simply put the component in the view

### Expected behavior

For example if the query param page is 2 the component selects 2

**Comment by j0rgedev** [comment]
Uhm, this seems interesting. I noticed there's no prop for setting the initial page, so we could consider adding one. I'll take a closer look at it tomorrow 👀
